ocaml/xapi/export.ml

```diff
@@ -328,7 +341,39 @@ let make_gpu_group table __context self =
 		snapshot = API.Legacy.To.gPU_group_t group
 	}
 
-let make_all ~with_snapshot_metadata ~preserve_power_state table __context = 
+let make_pvs_proxies table __context self =
+	debug "exporting PVS Proxy %s" (Ref.string_of self);
+	let lookup' ref = lookup table (Ref.string_of ref) in
+	let proxy = Db.PVS_proxy.get_record ~__context ~self in
+	let proxy =
+		{ proxy with
+		  API.pVS_proxy_site      = lookup' proxy.API.pVS_proxy_site
+		; API.pVS_proxy_VIF       = lookup' proxy.API.pVS_proxy_VIF
+		; API.pVS_proxy_cache_SR  = lookup' proxy.API.pVS_proxy_cache_SR
```

Comment:
I think we need to blank this field, because the site's cache SRs on the receiving pool are different in general.